web/template/supporter/organisation_details.gohtml

```diff
@@ -35,10 +35,15 @@
               <dt class="govuk-summary-list__key">
                 {{ .Organisation.Name }}
               </dt>
-              <dd class="govuk-summary-list__value">
-              </dd>
               <dd class="govuk-summary-list__actions">
-                <a class="govuk-link" href="{{ link .App global.Paths.Supporter.EditOrganisationName.Format }}">{{ tr .App "change" }}<span class="govuk-visually-hidden"> {{ tr .App "name" }}</span></a>
+                <ul class="govuk-summary-list__actions-list">
+                  <li class="govuk-summary-list__actions-list-item">
+                    <a class="govuk-link" href="{{ link .App global.Paths.Supporter.EditOrganisationName.Format }}">{{ tr .App "change" }}<span class="govuk-visually-hidden"> {{ tr .App "name" }}</span></a>
```

Comment:
this label should be "Change name" now

Reply:
also on the design there isn't a | between the items, but I don't know if that is actually preferable?

Reply:
Yeah I was going to leave with the `|` and see what Beth thinks at the next design review (we get it for free with that component so 🤷 )